ext/gd/libgd/gd_interpolation.c

```diff
@@ -2137,8 +2137,7 @@ gdImagePtr gdImageRotateInterpolated(const gdImagePtr src, const float angle, in
 	/* round to two decimals and keep the 100x multiplication to use it in the common square angles
 	   case later. Keep the two decimal precisions so smaller rotation steps can be done, useful for
 	   slow animations. */
-	const int angle_rounded = fmod((int) floorf(angle * 100), 360 * 100);
-
+	const int angle_rounded = (int) fmod((int) floorf(angle * 100), 360 * 100);
```

Comment:
This looks seriously overengineered. Why are there even float operations, instead of
```suggestion
	const int angle_rounded = ((int) (angle * 100)) % (360 * 100);
```